### ISSUE TITLE
[HW2] About author names of articles

### ISSUE THREAD

**Issue by 93lemondrops** [swsnu/swppfall2017#38]
On features, it is written that we should show author names of articles.

 - Articles list page (/articles)
   - Users should be able to clearly make out the followings:
     - `article` `id`s
     - `article` `title`s
     - `author` `name`s

But when we call `api/articles` API, we get `author_id` only, not author name. Does this mean we have to get author name by author id and show it?

**Comment by wonook** [comment]
Yes, that is the point! We access the information of other models by their id, and those information do not need to be replicated to other models.
